src/lib/y2storage/planned/can_be_encrypted.rb

```diff
@@ -130,11 +134,13 @@ def final_device!(plain_device)
           assign_enc_attr(result, :pbkdf)
           assign_enc_attr(result, :label)
           assign_enc_attr(result, :cipher)
+          assign_enc_attr(result, :authentication)
           assign_enc_attr(result, :key_size) { |value| value / 8 }
-          log.info "Device encrypted. Returning the new device #{result.inspect}"
+          log.info "Device encrypted. Returning a new device #{result.inspect}"
```

Comment:
Not sure if that change helps much. The message is not visible by the user and introducing cosmetic changes here can be a problem when trying to debug across versions. I prefer and not-so-nice but stable string than a polished but changing one.

Reply:
Yes, you are right. Reverted....